.github/metaprompts/triage.md

```
@@ -0,0 +1,147 @@
+- [ ] Is it stale (>7 days no activity)?
+
+**Check Quality:**
+- [ ] Has clear description
+- [ ] Has appropriate size
+- [ ] Has test coverage
+
+### 3. 🏷️ LABEL
+
+Apply labels based on analysis:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)